src/ashley/editor/decorators.py

```diff
@@ -0,0 +1,23 @@
+"""This module contains custom decorators for draftjs_exporter"""
+import logging
+
+from draftjs_exporter.dom import DOM
+
+logger = logging.getLogger("ashley")
+
+
+def link(props):
+    """Entity decorator for links"""
```

Comment:
explain what it is doing and why you need it?

Reply:
I added more comments, is it clearer ?